ui/src/shared/components/Crosshair.tsx

```diff
@@ -34,9 +34,17 @@ class Crosshair extends PureComponent<Props> {
   }
 
   private get isVisible() {
-    const {hoverTime} = this.props
+    const {dygraph, hoverTime} = this.props
+    const timeRanges = dygraph.xAxisRange()
+
+    const minTimeRange = timeRanges[0]
```

Comment:
will this always exist?

Reply:
From what I have seen. However, this may cause a problem when there are no results. I will add a check to make sure this doesn't cause errors if its empty.